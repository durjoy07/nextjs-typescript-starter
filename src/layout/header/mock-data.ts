import { IHeader } from './interface';

export const headerProps: IHeader = {
  logo: 'Lemon Hive',
  menuItems: [
    {
      label: 'Home',
      href: '/',
    },
    {
      label: 'About',
      href: '/',
    },
    {
      label: 'Service',
      href: '/',
    },
    {
      label: 'Blog',
      href: '/',
    },
  ],
};
