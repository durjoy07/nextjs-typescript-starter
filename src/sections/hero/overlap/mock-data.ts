import { IHeroWithOverlapContent } from './interface';

export const heroWithOverlapContentProps: IHeroWithOverlapContent = {
  title: 'Lemon Hive, We Build Lightning Fast Websites',
  description:
    'We Build Lightning Fast Websites With Amazing UX Using Headless Architectures',
  buttons: {
    buttonLeft: {
      label: 'Server Side Example',
      href: '/',
    },
    buttonRight: {
      label: 'Client Side Example',
      href: 'client-side',
    },
  },
};
