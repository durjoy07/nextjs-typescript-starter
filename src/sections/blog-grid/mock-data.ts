import { IBlogGrid } from './interface';
import { blogCardProps } from '@components/cards/blog/mock-data';
import { sectionDescriptionProps } from '@components/section-heading/mock-data';

export const blogGridProps: IBlogGrid = {
  sectionHeading: {
    title: 'Our Blog',
    description: sectionDescriptionProps,
  },
  blogs: [
    { ...blogCardProps, tags: ['Nextjs', 'Reactjs'] },
    { ...blogCardProps, tags: ['Nextjs', 'Reactjs'] },
    { ...blogCardProps, tags: ['Nextjs'] },
    { ...blogCardProps },
    { ...blogCardProps },
    { ...blogCardProps },
    { ...blogCardProps },
    { ...blogCardProps },
  ],
};
