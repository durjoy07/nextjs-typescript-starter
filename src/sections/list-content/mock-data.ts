import { sectionDescriptionProps } from '@components/section-heading/mock-data';
import { ISanityCustomPortableText } from '@root/src/backend/types/portable-text';
import { IListContentSection } from './interface';

export const descriptionPortableData: ISanityCustomPortableText = [
  {
    _key: 'd956675f69e0',
    children: [
      {
        _type: 'span',
        marks: [],
        text: 'For design, branding, web development, marketing, consulting & other agencies.',
        _key: '612b8d7ad65a0',
      },
    ],
    markDefs: [],
    _type: 'customBlock',
    style: 'normal',
  },
  {
    _key: 'f9a72086f01c',
    children: [
      {
        _type: 'span',
        marks: [],
        text: 'We work with agencies in a number of different ways, but predominantly as an extension of your team to add headless development expertise.',
        _key: '3ec390abb3560',
      },
    ],
    markDefs: [],
    _type: 'customBlock',
    style: 'normal',
  },
  {
    _key: 'af14faef2a19',
    children: [
      {
        _type: 'span',
        marks: [],
        text: 'With the move towards headless website development, it can be difficult to find the right people, with the right expertise, to produce websites to your standards. By outsourcing the development and/or design to us, we can ensure you build exceptional headless websites that exceed what is possible with monolithic platforms such as WordPress.',
        _key: '57de12dc89fc0',
      },
    ],
    markDefs: [],
    _type: 'customBlock',
    style: 'normal',
  },
];

export const listContentSectionProps: IListContentSection = {
  sectionTheme: 'secondary',
  sectionHeading: {
    title: 'Who We Work With',
    description: sectionDescriptionProps,
  },
  lists: [
    {
      title: 'Services for Agencies',
      description: descriptionPortableData,
    },
    {
      title: 'Services for Agencies',
      description: descriptionPortableData,
    },
  ],
};
