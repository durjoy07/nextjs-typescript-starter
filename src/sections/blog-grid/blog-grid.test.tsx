import { render } from '@testing-library/react';
import BlogGrid from '.';
import { blogGridProps } from './mock-data';

it('renders blog grid correctly', () => {
  const { getAllByRole } = render(<BlogGrid {...blogGridProps} />);
  const element = getAllByRole('presentation')[0];

  expect(element).toBeInTheDocument();
  expect(element).toMatchSnapshot();
});
