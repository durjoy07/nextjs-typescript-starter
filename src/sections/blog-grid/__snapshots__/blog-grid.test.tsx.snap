// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders blog grid correctly 1`] = `
<section
  class="section-padding-primary bg-gray-medium"
  role="presentation"
>
  <div
    class="container"
    role="presentation"
  >
    <div
      class="max-w-[700px]"
    >
      <div
        class="space-y-2 lg:space-y-3 mb-[1.875rem] md:mb-[3.25rem] section-heading primary"
        role="presentation"
      >
        <h2
          class="text-[1.75rem] font-medium leading-[1.3] md:text-[3rem] md:font-bold md:leading-[1.1] title"
        >
          Our Blog
        </h2>
        <div
          class="text-[.875rem] leading-[1.3] md:text-base mt-3 lg:mt-5 description"
        >
          <article
            class="prose max-w-full space-y-4"
          >
            <p
              class="dark:text-white"
              id="0ffdc2e0f0b8"
            >
              <a
                class="no-underline styled-link"
                href="https://www.google.com"
                target="_blank"
              >
                <span
                  class="link-underline"
                >
                  Headless website
                </span>
              </a>
               for an award winning digital marketing agency.
            </p>
            <br />
            <p
              class="dark:text-white"
              id="c0b3d76ed804"
            >
              <strong>
                Lorem ipsum
              </strong>
               dolor 
              <em>
                sit amet consectetur
              </em>
              . Vitae cursus ultricies interdu ridiculus. Nullam nuwel craegestas risus 
              <span
                style="text-decoration: underline;"
              >
                diam lorem nascetur
              </span>
              . Lorem ipsum dolor sit amet consectetur. Vitae cursus ultricies interdu ridiculus. Nullam nuwel craegestas risus 
              <strong>
                diam lorem nascetu ullam nuwel
              </strong>
            </p>
          </article>
        </div>
      </div>
    </div>
    <div
      class="grid grid-cols-1 gap-6 sm:grid-cols-2 xl:grid-cols-3 2xl:grid-cols-4"
    >
      <div
        class="relative overflow-hidden h-auto card shadow-one radius-one"
        role="presentation"
      >
        <a
          class="block"
          href="https://www.lemonhive.com/"
        >
          <div
            class="group/card relative h-full bg-success/[.2]"
          >
            <div
              class="relative"
            >
              <div
                class="overflow-hidden"
              >
                <img
                  alt="dummy image 2"
                  class="transition-transform ease-in-out group-hover/card:scale-110"
                  data-nimg="1"
                  decoding="async"
                  height="720"
                  loading="lazy"
                  src="http://localhost/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1080&q=75 1x, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75 2x"
                  style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
                  width="960"
                />
              </div>
            </div>
            <div
              class="flex flex-col space-y-3 p-5 pt-8"
            >
              <h3
                class="flex-1 text-lg font-bold line-clamp-2"
              >
                Headless Website Development
              </h3>
              <p
                class="line-clamp-3"
              >
                Headless solutions built with Sanity.io, Strapi, Prismic, Contentful and others with front-end frameworks such as Next.js, Gatsby, & Nuxt.
              </p>
              <div>
                <div
                  class="mt-4 flex items-center gap-2"
                >
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-warning"
                    role="presentation"
                  >
                    Nextjs
                  </span>
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-danger-light"
                    role="presentation"
                  >
                    Reactjs
                  </span>
                </div>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        class="relative overflow-hidden h-auto card shadow-one radius-one"
        role="presentation"
      >
        <a
          class="block"
          href="https://www.lemonhive.com/"
        >
          <div
            class="group/card relative h-full bg-success/[.2]"
          >
            <div
              class="relative"
            >
              <div
                class="overflow-hidden"
              >
                <img
                  alt="dummy image 2"
                  class="transition-transform ease-in-out group-hover/card:scale-110"
                  data-nimg="1"
                  decoding="async"
                  height="720"
                  loading="lazy"
                  src="http://localhost/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1080&q=75 1x, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75 2x"
                  style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
                  width="960"
                />
              </div>
            </div>
            <div
              class="flex flex-col space-y-3 p-5 pt-8"
            >
              <h3
                class="flex-1 text-lg font-bold line-clamp-2"
              >
                Headless Website Development
              </h3>
              <p
                class="line-clamp-3"
              >
                Headless solutions built with Sanity.io, Strapi, Prismic, Contentful and others with front-end frameworks such as Next.js, Gatsby, & Nuxt.
              </p>
              <div>
                <div
                  class="mt-4 flex items-center gap-2"
                >
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-warning"
                    role="presentation"
                  >
                    Nextjs
                  </span>
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-danger-light"
                    role="presentation"
                  >
                    Reactjs
                  </span>
                </div>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        class="relative overflow-hidden h-auto card shadow-one radius-one"
        role="presentation"
      >
        <a
          class="block"
          href="https://www.lemonhive.com/"
        >
          <div
            class="group/card relative h-full bg-success/[.2]"
          >
            <div
              class="relative"
            >
              <div
                class="overflow-hidden"
              >
                <img
                  alt="dummy image 2"
                  class="transition-transform ease-in-out group-hover/card:scale-110"
                  data-nimg="1"
                  decoding="async"
                  height="720"
                  loading="lazy"
                  src="http://localhost/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1080&q=75 1x, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75 2x"
                  style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
                  width="960"
                />
              </div>
            </div>
            <div
              class="flex flex-col space-y-3 p-5 pt-8"
            >
              <h3
                class="flex-1 text-lg font-bold line-clamp-2"
              >
                Headless Website Development
              </h3>
              <p
                class="line-clamp-3"
              >
                Headless solutions built with Sanity.io, Strapi, Prismic, Contentful and others with front-end frameworks such as Next.js, Gatsby, & Nuxt.
              </p>
              <div>
                <div
                  class="mt-4 flex items-center gap-2"
                >
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-warning"
                    role="presentation"
                  >
                    Nextjs
                  </span>
                </div>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        class="relative overflow-hidden h-auto card shadow-one radius-one"
        role="presentation"
      >
        <a
          class="block"
          href="https://www.lemonhive.com/"
        >
          <div
            class="group/card relative h-full bg-success/[.2]"
          >
            <div
              class="relative"
            >
              <div
                class="overflow-hidden"
              >
                <img
                  alt="dummy image 2"
                  class="transition-transform ease-in-out group-hover/card:scale-110"
                  data-nimg="1"
                  decoding="async"
                  height="720"
                  loading="lazy"
                  src="http://localhost/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1080&q=75 1x, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75 2x"
                  style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
                  width="960"
                />
              </div>
            </div>
            <div
              class="flex flex-col space-y-3 p-5 pt-8"
            >
              <h3
                class="flex-1 text-lg font-bold line-clamp-2"
              >
                Headless Website Development
              </h3>
              <p
                class="line-clamp-3"
              >
                Headless solutions built with Sanity.io, Strapi, Prismic, Contentful and others with front-end frameworks such as Next.js, Gatsby, & Nuxt.
              </p>
              <div>
                <div
                  class="mt-4 flex items-center gap-2"
                >
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-warning"
                    role="presentation"
                  >
                    Reactjs
                  </span>
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-danger-light"
                    role="presentation"
                  >
                    Nextjs
                  </span>
                </div>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        class="relative overflow-hidden h-auto card shadow-one radius-one"
        role="presentation"
      >
        <a
          class="block"
          href="https://www.lemonhive.com/"
        >
          <div
            class="group/card relative h-full bg-success/[.2]"
          >
            <div
              class="relative"
            >
              <div
                class="overflow-hidden"
              >
                <img
                  alt="dummy image 2"
                  class="transition-transform ease-in-out group-hover/card:scale-110"
                  data-nimg="1"
                  decoding="async"
                  height="720"
                  loading="lazy"
                  src="http://localhost/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1080&q=75 1x, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75 2x"
                  style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
                  width="960"
                />
              </div>
            </div>
            <div
              class="flex flex-col space-y-3 p-5 pt-8"
            >
              <h3
                class="flex-1 text-lg font-bold line-clamp-2"
              >
                Headless Website Development
              </h3>
              <p
                class="line-clamp-3"
              >
                Headless solutions built with Sanity.io, Strapi, Prismic, Contentful and others with front-end frameworks such as Next.js, Gatsby, & Nuxt.
              </p>
              <div>
                <div
                  class="mt-4 flex items-center gap-2"
                >
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-warning"
                    role="presentation"
                  >
                    Reactjs
                  </span>
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-danger-light"
                    role="presentation"
                  >
                    Nextjs
                  </span>
                </div>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        class="relative overflow-hidden h-auto card shadow-one radius-one"
        role="presentation"
      >
        <a
          class="block"
          href="https://www.lemonhive.com/"
        >
          <div
            class="group/card relative h-full bg-success/[.2]"
          >
            <div
              class="relative"
            >
              <div
                class="overflow-hidden"
              >
                <img
                  alt="dummy image 2"
                  class="transition-transform ease-in-out group-hover/card:scale-110"
                  data-nimg="1"
                  decoding="async"
                  height="720"
                  loading="lazy"
                  src="http://localhost/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1080&q=75 1x, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75 2x"
                  style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
                  width="960"
                />
              </div>
            </div>
            <div
              class="flex flex-col space-y-3 p-5 pt-8"
            >
              <h3
                class="flex-1 text-lg font-bold line-clamp-2"
              >
                Headless Website Development
              </h3>
              <p
                class="line-clamp-3"
              >
                Headless solutions built with Sanity.io, Strapi, Prismic, Contentful and others with front-end frameworks such as Next.js, Gatsby, & Nuxt.
              </p>
              <div>
                <div
                  class="mt-4 flex items-center gap-2"
                >
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-warning"
                    role="presentation"
                  >
                    Reactjs
                  </span>
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-danger-light"
                    role="presentation"
                  >
                    Nextjs
                  </span>
                </div>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        class="relative overflow-hidden h-auto card shadow-one radius-one"
        role="presentation"
      >
        <a
          class="block"
          href="https://www.lemonhive.com/"
        >
          <div
            class="group/card relative h-full bg-success/[.2]"
          >
            <div
              class="relative"
            >
              <div
                class="overflow-hidden"
              >
                <img
                  alt="dummy image 2"
                  class="transition-transform ease-in-out group-hover/card:scale-110"
                  data-nimg="1"
                  decoding="async"
                  height="720"
                  loading="lazy"
                  src="http://localhost/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1080&q=75 1x, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75 2x"
                  style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
                  width="960"
                />
              </div>
            </div>
            <div
              class="flex flex-col space-y-3 p-5 pt-8"
            >
              <h3
                class="flex-1 text-lg font-bold line-clamp-2"
              >
                Headless Website Development
              </h3>
              <p
                class="line-clamp-3"
              >
                Headless solutions built with Sanity.io, Strapi, Prismic, Contentful and others with front-end frameworks such as Next.js, Gatsby, & Nuxt.
              </p>
              <div>
                <div
                  class="mt-4 flex items-center gap-2"
                >
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-warning"
                    role="presentation"
                  >
                    Reactjs
                  </span>
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-danger-light"
                    role="presentation"
                  >
                    Nextjs
                  </span>
                </div>
              </div>
            </div>
          </div>
        </a>
      </div>
      <div
        class="relative overflow-hidden h-auto card shadow-one radius-one"
        role="presentation"
      >
        <a
          class="block"
          href="https://www.lemonhive.com/"
        >
          <div
            class="group/card relative h-full bg-success/[.2]"
          >
            <div
              class="relative"
            >
              <div
                class="overflow-hidden"
              >
                <img
                  alt="dummy image 2"
                  class="transition-transform ease-in-out group-hover/card:scale-110"
                  data-nimg="1"
                  decoding="async"
                  height="720"
                  loading="lazy"
                  src="http://localhost/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75"
                  srcset="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1080&q=75 1x, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdktw59lc2%2Fimage%2Fupload%2Fv1669267967%2FBrightonSEO%2FCarousel%2Fimage-2_yph8q6.jpg&w=1920&q=75 2x"
                  style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
                  width="960"
                />
              </div>
            </div>
            <div
              class="flex flex-col space-y-3 p-5 pt-8"
            >
              <h3
                class="flex-1 text-lg font-bold line-clamp-2"
              >
                Headless Website Development
              </h3>
              <p
                class="line-clamp-3"
              >
                Headless solutions built with Sanity.io, Strapi, Prismic, Contentful and others with front-end frameworks such as Next.js, Gatsby, & Nuxt.
              </p>
              <div>
                <div
                  class="mt-4 flex items-center gap-2"
                >
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-warning"
                    role="presentation"
                  >
                    Reactjs
                  </span>
                  <span
                    class="inline-flex items-center justify-center px-3 py-1 text-sm font-semibold text-black rounded-full bg-danger-light"
                    role="presentation"
                  >
                    Nextjs
                  </span>
                </div>
              </div>
            </div>
          </div>
        </a>
      </div>
    </div>
  </div>
</section>
`;
